src/client/components/pages/Courses/MeetingModal.tsx

```diff
@@ -266,6 +333,17 @@ const MeetingModal: FunctionComponent<MeetingModalProps> = function ({
     setNewMeetingIdNumber(newMeetingIdNumber + 1);
   };
 
+  /**
+   * Calls the onChange function to indicate that there has been a change in the
+   * meetings of the current modal. If the user exits the modal without saving,
+   * there will be a warning message to prevent the user from losing changes.
+   */
+  const signalModalChange = () => {
+    if (!isChanged) {
+      setIsChanged(true);
+    }
+  };
```

Comment:
Could we just call `setIsChanged(true);` in place of `signalModalChange` on lines 356 and 469 below? I don't think it should trigger any effect hooks to set the state value to the same value....

Reply:
Sure thing! I originally had the `signalModalChange` function to try to have as many reusable parts as possible for the others modals and also to try not to make calls to update the state unless necessary, but this seems simple enough to not be in a function.